charts/osm/values.yaml

```diff
@@ -65,6 +65,20 @@ OpenServiceMesh:
       # -- Prometheus data retention time
       time: 15d
 
+  #
+  # -- kube-state-metrics parameters
+  kubeStateMetrics:
+    # -- kube-state-metrics container resource parameters
+    resources:
+      limits:
+        cpu: "1"
+        memory: "2G"
```

Comment:
Why does this require such a high memory limit? Was this profiled? I'd be inclined to set both the limits and requests memory to a much lower value (64M).

Reply:
I was hoping to get feedback on the resource limit values. I wasn't sure what values would be appropriate for kube-state-metrics.

Reply:
@eduser25, @nojnhuh can share details on how to profile this. I would start with a lower cpu and memory value and profile the process to understand the average/max values for these properties.